app/assets/javascripts/components/MessageForm.js.jsx.coffee

```diff
@@ -0,0 +1,30 @@
+@MessageForm = React.createClass
+  setAuthor: ->
+    userActions.setAuthor(React.findDOMNode(this).elements.author.value)
+
+  onChangeText: (e) ->
+    @setAuthor()
+    userActions.typing()
+
+  onSubmit: (e) ->
+    userActions.stopTyping()
+    e.preventDefault()
+    form = e.target
+
+    data =
+      text: form.elements.text.value
+      author: form.elements.author.value
+
+    # trigger the action
+    userActions.sendMessage data
```

Comment:
In short, yes.  It will trigger the `userActions.sendMessage` action, and the `messageStore` is listening to all of the `userActions` because it uses `@listenToMany(userActions)`